app/src/main/java/com/cooper/wheellog/WheelData.java

```diff
@@ -1,12 +1,15 @@
 package com.cooper.wheellog;
 
+import android.app.Activity;
```

Comment:
Are you sure that it is needed?
import android.app.Activity;
import android.app.Application;

Reply:
Probably not. Will check.